### PR TITLE
Bt 11 final delete

### PR DESCRIPTION
**Pull request by Neethujn**
updated code for delete Job title

### CODE REVIEW

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    private final String jobs=" //div[contains(@class, 'oxd-table-body')]/div[contains(@class,'oxd-table-card')]";
+
+
+    @FindBy(xpath = jobs)
+    private List<WebElement> listJobs;
+
+    public JobTitlePage(WebDriver driver) {
+        super(driver);
+        this.driver=driver;
+
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: remove unnecessary space

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        Assert.assertFalse(match);
+        System.out.println("\n");
+        System.out.println(jobName + " is deleted successfully");
+        System.out.println("\n");
+        }
+
+    public void clickDeleteIcon(String jname){
+        String btnxpath="//div[@class='oxd-table-row oxd-table-row--with-border' and .//div[text()='"+jname+"']]//button[1]";
+        click(By.xpath(btnxpath));
+        click(By.xpath("//*[@id='app']/div[3]/div/div/div/div[3]/button[2]"));
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: move this up with variables

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                break;
+            }
+        }
+        Boolean match = listJobs.stream().map(s -> s.getText()).anyMatch(s -> s.equalsIgnoreCase(newjob));
+        Assert.assertTrue(match);
+        System.out.println("\n");
+        System.out.println(newjob + " is updated successfully");
+        System.out.println("\n");
+    }
+    public void clickEditAction(String oldjobName,String newjob){
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: delete this method

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.JOB_TITLES);
+        JobTitlePage jobPage = new JobTitlePage(driver);
+        jobPage.deleteAndVerifyJob(jobName);
+
+    }
+    @Test
+    public  void updateJob(){
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: remove update

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java (hunk 1 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: Done with the changes

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java (hunk 3 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: Deleted the Method

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java (hunk 4 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/JobTitleTest.java]: removed the update job codes

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java (hunk 2 above)
Comment (Neethujn) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/JobTitlePage.java]: moved the variable to the top.

**Review by RahulNair92 — APPROVED** [review]

**Pull request merged**